CMakeLists.txt

```diff
@@ -291,6 +293,19 @@ if(SENTRY_TRANSPORT_CURL)
 	endif()
 endif()
 
+if(SENTRY_TRANSPORT_COMPRESSION)
+	if(NOT ZLIB_FOUND)
+		find_package(ZLIB REQUIRED)
+	endif()
+
+	if(SENTRY_BACKEND_CRASHPAD)
+		set(CRASHPAD_ZLIB_SYSTEM ON CACHE BOOL "Force CRASHPAD_ZLIB_SYSTEM when enabling transport compression" FORCE)
```

Comment:
If we set this here, we need to ensure that `find_package()` in the crashpad `CMakeLists.txt` is only invoked if `NOT ZLIB_FOUND` as we do above:

https://github.com/getsentry/crashpad/blob/58d7207c3c5fac96bbbca8e760824714ec3f8a29/CMakeLists.txt#L25

Reply:
If you are unsure how to apply this change via submodules and separate PR to the crashpad fork, I can also do this.

Reply:
Yes, I am not familiar with the zip dependency relationship between `crashpad` submodules and `sentry-native`, so I removed this line(`set(CRASHPAD_ZLIB_SYSTEM...`)

Reply:
Okay, no worries, I can take care of it.